.github/workflows/publish-candidate.yaml

```diff
@@ -68,7 +68,8 @@ jobs:
           release="--release=edge --release=candidate"
 
           charmcraft upload $release slurmctld$extension --resource etcd:1
+          charmcraft upload $release slurmd$extension --resource nhc:1
 
-          for charm in slurmd slurmdbd slurmrestd; do
-              charmcraft upload "$release" "$charm$extension"
+          for charm in slurmdbd slurmrestd; do
+              charmcraft upload $release "$charm$extension"
```

Comment:
Please also apply these changes to the github/workflows/publish-edge.yaml file

Reply:
Done in [d351c2b](https://github.com/omnivector-solutions/slurm-charms/pull/151/commits/d351c2b7a57f0b1001924732b018b4abbda39c78)